pvr.hts/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v20.3.1
```

Comment:
Sorry, for incompatible API bumps, add-ons minor version number needs to be increased, not micro. Micro is for bug fixes only, which keep API compatibility and add no new functionality.

This holds not only for pvr.hts, but for any binary add-on. So please also adjust the other PRs.